blocks/users-list/edit.js

```diff
@@ -1,3 +1,36 @@
-export default function Edit() {
-	return <div> Hello from edit </div>;
+import ServerSideRender from '@wordpress/server-side-render';
+import { useSelect } from '@wordpress/data';
+
+export default function Edit( { attributes, setAttributes } ) {
+	const taxonomies = useSelect( ( select ) => {
+		/* TODO: Change taxonomy name */
+		return select( 'core' ).getEntityRecords( 'taxonomy', 'staff_position' );
```

Comment:
The taxonomy name is `group`

https://github.com/creativecommons/wp-plugin-creativecommons-website/blob/00a6a5323c7439cbab9ba445a9c27d1741728d66/cc-author/cc-author-group-taxonomy.php#L9-L11

Reply:
What does `select( 'core' )` do here? It seems like `staff_position` is the taxonomy name, correct?

```suggestion
		return select( 'core' ).getEntityRecords( 'taxonomy', 'group' );
```